admin/manifest-dev.yml

```diff
@@ -19,4 +19,5 @@ applications:
     env:
       <<: *defaults_env
       ADMIN_BASE_URL: https://notify-((stg)).apps.y.cld.gov.au
+      DOCS_BASE_URL: https://notify-docs-((stg)).apps.y.cld.gov.au/
```

Comment:
I think you need to remove the trailing slash here: 

![image](https://user-images.githubusercontent.com/626664/47825340-75d52980-ddc5-11e8-92c9-f66bc8ad47c1.png)